configureWorkspace/configure.ts

```diff
@@ -53,11 +53,18 @@ export type ConfigureTelemetryProperties = {
     packageFileSubfolderDepth?: string; // 0 = project/etc file in root folder, 1 = in subfolder, 2 = in subfolder of subfolder, etc.
 };
 
-const generatorsByPlatform = new Map<Platform, {
+export interface IPlatformGeneratorInfo {
     genDockerFile: GeneratorFunction,
     genDockerCompose: GeneratorFunction,
-    genDockerComposeDebug: GeneratorFunction
-}>();
+    genDockerComposeDebug: GeneratorFunction,
+    defaultPort: string
+}
+
+export function getExposeStatements(port: string): string {
```

Comment:
getExposeStatement? I don't see the reasoning behind the plural name.

Reply:
VS has scenarios where there are more than one (HTTPS and HTTP ports).  I'm trying to keep similar to their logic.